models/Circle.js

```diff
@@ -6,22 +6,22 @@ const Circle = sequelize.define('circle', {
         allowNull: false,
         defaultValue: '',
```

Comment:
The defaultValue is the empty string, which is invalid value.

Reply:
Yeah, I just replaced it as a first step. But I'll go through the defaultValues as well now. Thanks for the comment!

Reply:
Fixed